client/src/App.tsx

```diff
@@ -73,14 +74,16 @@ export const App = () => {
                     <div className="text-lg">Plugins</div>
                   </Link>
                 </li>
-                <li>
-                  <a href="https://discord.gg/k7C8kp3b76" target="_blank" rel="noreferrer" className="text-lg">
-                    <div className="flex">
-                      <img src="/discord.svg" className="w-6 pr-1" alt="Discord" />
-                      Discord
-                    </div>
-                  </a>
-                </li>
+                <Feature flag="useIQEngineOutReach">
```

Comment:
would be good to add a component test to test this works when: flag is null, flag is set to true and flag is set to null

Reply:
Also, if feature flag is null, it should be set to true